package.json

```diff
@@ -48,9 +49,13 @@
     "ember-try": "^0.2.23",
     "eslint-plugin-ember": "^5.0.0",
     "eslint-plugin-node": "^6.0.1",
-    "loader.js": "^4.2.3"
+    "esm": "^3.0.63",
+    "loader.js": "^4.2.3",
+    "qunit": "^2.6.1"
   },
+  "main": "addon/index.js",
```

Comment:
Hmm, I think you need to follow the normal esm suggested path for this to work.

* add a `main.js` that does:

```js
// Set options as a parameter, environment variable, or rc file.
require = require("esm")(module/*, options*/);
module.exports = require("./addon");
```

* move `esm` to a `dependency`
* change tests to not use `esm` directly